arcaflow_plugin_coremark_pro/coremark_pro_schema.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import typing
+from dataclasses import dataclass
+from arcaflow_plugin_sdk import schema, plugin
+
+
+@dataclass
+class CertifyAllParams:
+    contexts: typing.Annotated[
+        int,
+        schema.name("Contexts"),
+        schema.description("Number of contexts per benchmark"),
+    ]
+    workers: typing.Annotated[
+        int,
+        schema.name("Workers"),
+        schema.description("Number of workers per benchmark"),
+    ]
+
+
+@dataclass
+class CertifyAllItem:
+    multi_core: typing.Annotated[
+        float,
+        schema.id("MultiCore"),
+        schema.name("MultiCore"),
+        schema.description("Workload retult - Multi-Core (iter/s)"),
+    ]
+    single_core: typing.Annotated[
+        float,
+        schema.id("SingleCore"),
+        schema.name("SingleCore"),
+        schema.description("Workload retult - Single-Core (iter/s)"),
+    ]
+    scaling: typing.Annotated[
+        float,
+        schema.id("Scaling"),
+        schema.name("Scaling"),
+        schema.description("Workload retult - Scaling"),
+    ]
+
+
+@dataclass
+class CertifyAllResult:
+    cjpeg_rose7_preset: typing.Annotated[
+        CertifyAllItem,
+        schema.id("cjpeg-rose7-preset"),
+        schema.name("CJPEG Rose7 Preset"),
+        schema.description("Workload: CJPEG Rose7 Preset"),
+    ]
+    core: typing.Annotated[
+        CertifyAllItem,
+        schema.id("core"),
+        schema.name("Core"),
+        schema.description("Workload: Core"),
+    ]
+    linear_alg_mid_100x100_sp: typing.Annotated[
+        CertifyAllItem,
+        schema.id("linear_alg-mid-100x100-sp"),
+        schema.name("Linear Alg Mid 100x100 SP"),
+        schema.description("Workload: Linear Alg Mid 100x100 SP"),
+    ]
+    loops_all_mid_10k_sp: typing.Annotated[
+        CertifyAllItem,
+        schema.id("loops-all-mid-10k-sp"),
+        schema.name("Loops All Mid 10k SP"),
+        schema.description("Workload: Loops All Mid 10k SP"),
+    ]
+    nnet_test: typing.Annotated[
+        CertifyAllItem,
+        schema.name("NNet Test"),
+        schema.description("Workload: NNet Test"),
```

Comment:
Should there be an `id` property?

Reply:
Nope. It's only needed in the case that the parameter has hyphens or other characters that can't be included in the object name in the class.